packages/camera/camera_avfoundation/example/ios/RunnerTests/Mocks/MockFLTCam.swift

```diff
@@ -2,207 +2,207 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-import camera_avfoundation
+@testable import camera_avfoundation
 
 // Import Objectice-C part of the implementation when SwiftPM is used.
 #if canImport(camera_avfoundation_objc)
-  import camera_avfoundation_objc
+  @testable import camera_avfoundation_objc
```

Comment:
curious does `@testable import` have any effect on objc modules?

Reply:
Probably not, TBH I haven't considered it. I will keep this one for consistency and we will be able to remove all of them shortly